src/Sulu/Component/Rest/ListBuilder/Metadata/ListXmlLoader.php

```diff
@@ -129,8 +129,14 @@ private function loadPropertyMetadata(\DOMXPath $xpath, \DOMNode $propertyNode)
         $propertyMetadata->setSortable(
             XmlUtil::getBooleanValueFromXPath('@sortable', $xpath, $propertyNode, true)
         );
-        $propertyMetadata->setFilterType(XmlUtil::getValueFromXPath('@filter-type', $xpath, $propertyNode));
-        $propertyMetadata->setFilterTypeParameters($this->getFilterTypeParameters($xpath, $propertyNode));
```

Comment:
Not sure if we should introduce a new `FilterTypeMetadata` class instead of having two prefixed properties on that `PropertyMetadata` class. But somehow it feels like not big enough of a deal to accept that BC break...

Reply:
depends if we plan to support multiple filter-types per property in the future. if yes, i would probably make this BC break now

Reply:
also if we expect that the `filter` node will get additional attributes in the future

Reply:
In that case I would say we don't do it. Defining multiple filters would break our entire UI concept, so that probably won't happen before a major release anyway, resp. only by using some kind of BC layer.